### PR TITLE
remove members who have left VMware

### PR DESCRIPTION
**Pull request by anita-flegg**
Bob G and Lora B were contractors with the VMware TAS group, and they both left when Broadcom took over.

### CODE REVIEW

**Comment by beyhan** [comment]
According to our [Revoking Reviewer or Approver Role](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#revoking-reviewer-or-approver-role) @bobbygeeze and @blora1 have two weeks to refute the revocation.

**Comment by beyhan** [comment]
@anita-flegg this can be merged.

**Comment by anita-flegg** [comment]
Hi @ameowlia, would you approve this when you get a chance? Thanks!

**Review by beyhan — APPROVED** [review]

**Pull request merged**
